tests/queries/0_stateless/03005_first_day_of_week.sql

```diff
@@ -0,0 +1,50 @@
+SELECT
```

Comment:
A future reader will have a hard time figuring out what this test is supposed to test.

Please add a comment on top that it is about setting `first_day_of_week`.

Also, the three statements each test too much stuff. Please break that further down, ideally each SELECT tests a single affected SQL function (e.g. `dateDiff`).